Core/Rewriters/MoqSyntaxFactory.cs

```diff
@@ -301,5 +301,41 @@ public static class MoqSyntaxFactory
                     SyntaxKind.SimpleMemberAccessExpression,
                     SyntaxFactory.IdentifierName (identifierName),
                     SyntaxFactory.IdentifierName ("Verify"))));
+
+    public static FieldDeclarationSyntax MockFieldDeclaration (
+        SyntaxList<AttributeListSyntax> attributeList,
+        SyntaxTokenList modifiers,
+        TypeSyntax declarationType,
+        VariableDeclaratorSyntax variableDeclarator) =>
+        SyntaxFactory.FieldDeclaration (
+            attributeList,
+            modifiers,
+            SyntaxFactory.VariableDeclaration (
+                    SyntaxFactory.GenericName (SyntaxFactory.Identifier ("Mock"))
+                        .WithTypeArgumentList (
+                            SyntaxFactory.TypeArgumentList (
+                                SyntaxFactory.SingletonSeparatedList (
+                                    declarationType))))
+                .WithVariables (
+                    SyntaxFactory.SingletonSeparatedList (
+                        variableDeclarator
+                            .WithLeadingTrivia (SyntaxFactory.Space))));
+
+    public static ArgumentSyntax MockObjectArgument (IdentifierNameSyntax identifierName) =>
+        SyntaxFactory.Argument (
+            SyntaxFactory.MemberAccessExpression (
+                    SyntaxKind.SimpleMemberAccessExpression,
+                    identifierName,
+                    SyntaxFactory.IdentifierName ("Object"))
+                .WithOperatorToken (
+                    SyntaxFactory.Token (SyntaxKind.DotToken)));
```

Comment:
Expression-bodied methods should not be used if the expression spans multiple lines if I remember correctly.